### ISSUE TITLE
Compiling parking_lot v0.10.2 error: cannot find macro `llvm_asm` in this scope

### ISSUE THREAD

**Issue by matrem94** [PyO3/pyo3#878]
## 🐛 Bug Reports

I'm trying to build the example found on [crates.io](https://crates.io/crates/pyo3). However, during compilation, I get the following error 

```rust
   Compiling parking_lot v0.10.2
error: cannot find macro `llvm_asm` in this scope                      ] 30/51: syn, regex, regex-syntax, parking_lot
  --> /Users/maximetremblay/.cargo/registry/src/github.com-1ecc6299db9ec823/parking_lot-0.10.2/src/elision.rs:77:13
   |
77 |             llvm_asm!("xacquire; lock; cmpxchgq $2, $1"
   |             ^^^^^^^^


error: cannot find macro `llvm_asm` in this scope
   --> /Users/maximetremblay/.cargo/registry/src/github.com-1ecc6299db9ec823/parking_lot-0.10.2/src/elision.rs:108:13
    |
108 |             llvm_asm!("xrelease; lock; xaddq $2, $1"
    |             ^^^^^^^^


error: aborting due to 2 previous errors======>                        ] 30/51: syn, regex, regex-syntax, parking_lot


error: could not compile `parking_lot`.
```

### 🌍 Environment

 - Your operating system and version: **MacOs Mojave 10.14.6**
 - Your python version: **python = 2.7.16, python3 = 3.7.4**
 - How did you install python (e.g. apt or pyenv)? Did you use a virtualenv?: `brew install python`
 - Your rust version (`rustc --version`): **rustc 1.44.0-nightly (f509b26a7 2020-03-18)**
 - Are you using the latest pyo3 version? Have you tried using latest master (replace `version = "0.x.y"` with `git = "https://github.com/PyO3/pyo3")?` **I'm using `0.9.2` and it is not working. However, I just tried using git instead of version and it worked.**

### 💥 Reproducing

I tried to build to project by using `cargo build --release` and `maturin build` and both failed for the same reason. I created a [git repo](https://github.com/matrem94/trying-pyo3) with the code.

**Comment by davidhewitt** [comment]
Hi @matrem94 you're correct; we've fixed this issue on the git master. We will release this fix with `0.10.0` if you're happy to use the git version for now.

**Comment by matrem94** [comment]
Hi @davidhewitt. This if fine for me. After I notice the quick fix, I kept my report just in case you didn't know. Thanks for the answer.
